Specs/Scene/ModelExperimental/ModelExperimentalAnimationCollectionSpec.js

```diff
@@ -615,6 +616,100 @@ describe("Scene/ModelExperimental/ModelExperimentalAnimationCollection", functio
     });
   });
 
+  it("animates with an explicit animation time", function () {
+    return loadAndZoomToModelExperimental(
+      {
+        gltf: animatedTriangleUrl,
+      },
+      scene
+    ).then(function (model) {
+      const time = JulianDate.fromDate(
+        new Date("January 1, 2014 12:00:00 UTC")
+      );
+      const animationCollection = model.activeAnimations;
+      animationCollection.animateWhilePaused = false;
+      let animationTime = 0;
+      const animation = animationCollection.add({
+        index: 0,
+        animationTime: function (duration) {
+          return animationTime / duration;
+        },
+      });
+
+      const spyUpdate = jasmine.createSpy("listener");
+      animation.update.addEventListener(spyUpdate);
+
+      scene.renderForSpecs(time);
+      animationTime = 0.1;
+      scene.renderForSpecs(JulianDate.addSeconds(time, 1.0, scratchJulianDate));
+      scene.renderForSpecs(JulianDate.addSeconds(time, 2.0, scratchJulianDate));
+      animationTime = 0.2;
+      scene.renderForSpecs(JulianDate.addSeconds(time, 2.0, scratchJulianDate));
+      animationTime = 0.3;
+      scene.renderForSpecs(JulianDate.addSeconds(time, 3.0, new JulianDate()));
+
+      expect(spyUpdate.calls.count()).toEqual(3);
+      expect(spyUpdate.calls.argsFor(0)[2]).toEqualEpsilon(
+        0.0,
+        CesiumMath.EPSILON14
+      );
+      expect(spyUpdate.calls.argsFor(1)[2]).toEqualEpsilon(
+        0.1,
+        CesiumMath.EPSILON14
+      );
+      expect(spyUpdate.calls.argsFor(2)[2]).toEqualEpsilon(
+        0.3,
+        CesiumMath.EPSILON14
+      );
+    });
+  });
+
+  it("animates while paused with an explicit animation time", function () {
+    return loadAndZoomToModelExperimental(
+      {
+        gltf: animatedTriangleUrl,
+      },
+      scene
+    ).then(function (model) {
+      const time = JulianDate.fromDate(
+        new Date("January 1, 2014 12:00:00 UTC")
+      );
+      const animationCollection = model.activeAnimations;
+      animationCollection.animateWhilePaused = true;
+      let animationTime = 0;
+      const animation = animationCollection.add({
+        index: 0,
+        animationTime: function (duration) {
+          return animationTime / duration;
+        },
+      });
+
+      const spyUpdate = jasmine.createSpy("listener");
+      animation.update.addEventListener(spyUpdate);
+
+      scene.renderForSpecs(time);
+      animationTime = 0.1;
+      scene.renderForSpecs(time);
+      scene.renderForSpecs(time);
```

Comment:
Add a comment here that this shouldn't update the animation because animationTime didn't change